docs/ref/contrib/admin/index.txt

```diff
@@ -2861,6 +2861,43 @@ Templates can override or extend base admin templates as described in
 ``AdminSite`` methods
 ---------------------
 
+.. method:: AdminSite.get_app_list(request, app_label=None)
+
+    .. versionadded:: 4.1
+
+    Returns a list of applications from the :doc:`application registry
+    </ref/applications/>` available for the current user. Each entry in the
+    list is a dict representing an application with the following keys:
```

Comment:
```suggestion
    list is a dictionary representing an application with the following keys:
```

Reply:
No need, we try to avoid trivial wording changes unless it's a big problem as it creates extra work for documentation translators.